rllib/algorithms/td3/tests/test_td3.py

```diff
@@ -54,6 +54,7 @@ def test_td3_exploration_and_with_random_prerun(self):
 
         # Test against all frameworks.
         for _ in framework_iterator(config, with_eager_tracing=True):
+            config.seed = 42
```

Comment:
I think we should avoid seeding wherever possible. Maybe others differ!

Reply:
Yeah, I agree. This seed here has been around for quite some time, though (I think this line was just move here).
So, ok to leave for now.